From 4cbc4fd790e99cfd2f414a08dd78a767bcdbbaca Mon Sep 17 00:00:00 2001
From: Nagaraj S <sivasankaranx.nagaraj@intel.com>
Date: Thu, 17 Mar 2016 13:50:53 -0700
Subject: [PATCH 365/441] Update Dual BSD/GPL in Linux/ce_mailbox.h

Change-Id: I83a5408bf1f116469aba2b34d7375eda507e21ce
IssueID:   HSD# 246053
IssueType: Defect
Symptom:   licensing issue in ce_mailbox.h files.
RootCause: Need to apply Dual BSD/GPL licensing.
Resolution: Added Dual BSD/GPL licensing.
Test:      Ran a test to verify boot.
Reviewer:  Warden Brett T and Strelchun, Timothy
ReviewID:  CCID-131895
---
 include/linux/ce_mailbox.h | 41 ++++++++++++++++++++++++++++++++++++++++-
 1 file changed, 40 insertions(+), 1 deletion(-)

--- a/include/linux/ce_mailbox.h
+++ b/include/linux/ce_mailbox.h
@@ -1,9 +1,14 @@
 /*
  *  include/linux/ce_mailbox.h
+ *  Description:
+ *  Contains RPC configuration api for the Intel(R) Puma(TM) 7 SoC family.
+ *
+ *  This file is provided under a dual BSD/GPLv2 license.  When using or
+ *  redistributing this file, you may do so under either license.
  *
  *  GPL LICENSE SUMMARY
  *
- *  Copyright(c) 2015 Intel Corporation. All rights reserved.
+ *  Copyright(c) 2015-2016 Intel Corporation. All rights reserved.
  *
  *  This program is free software; you can redistribute it and/or modify
  *  it under the terms of version 2 of the GNU General Public License as
@@ -20,8 +25,42 @@
  *  The full GNU General Public License is included in this distribution
  *  in the file called LICENSE.GPL.
  *
- *
+ *  Contact Information:
+ *  Intel Corporation
+ *  2200 Mission College Blvd.
+ *  Santa Clara, CA  97052
+ *
+ *  BSD LICENSE
+ *
+ *  Copyright(c) 2015-2016 Intel Corporation. All rights reserved.
+ *
+ *  Redistribution and use in source and binary forms, with or without
+ *  modification, are permitted provided that the following conditions
+ *  are met:
+ *
+ *   * Redistributions of source code must retain the above copyright
+ *     notice, this list of conditions and the following disclaimer.
+ *   * Redistributions in binary form must reproduce the above copyright
+ *     notice, this list of conditions and the following disclaimer in
+ *     the documentation and/or other materials provided with the
+ *     distribution.
+ *   * Neither the name Intel Corporation nor the names of its
+ *     contributors may be used to endorse or promote products derived
+ *     from this software without specific prior written permission.
+ *
+ *  THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
+ *  "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT
+ *  LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR
+ *  A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT
+ *  OWNER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL,
+ *  SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT
+ *  LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE,
+ *  DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY
+ *  THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+ *  (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+ *  OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  */
+
 #ifndef _CE_MBX_H_
 #define _CE_MBX_H_
 
